.eslintrc

```diff
@@ -6,7 +6,7 @@
     "es6": true
   },
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 8,
```

Comment:
ES8 is eslint's shorthand for ES2017.  No, you can't use "2017"...  Just subtract 2009. 😛